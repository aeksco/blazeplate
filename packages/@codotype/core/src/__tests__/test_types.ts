import { DEFAULT_ATTRIBUTE } from "../index";

test("testing @codotype/core", () => {
    expect(DEFAULT_ATTRIBUTE).toMatchSnapshot();
});
