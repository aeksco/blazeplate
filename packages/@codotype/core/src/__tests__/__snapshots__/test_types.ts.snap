// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing @codotype/core 1`] = `
Object {
  "addons": Object {},
  "datatype": null,
  "defaultValue": null,
  "id": "",
  "identifiers": Object {
    "camel": "",
    "kebab": "",
    "label": "",
    "pascal": "",
    "snake": "",
  },
  "internalNote": "",
  "locked": false,
  "source": "user",
}
`;
